com.adsbynimbus.nimbus/Editor/IOSPostBuildProcessor.cs

```diff
@@ -34,7 +35,7 @@ private static void PostProcessBuild_iOS(BuildTarget target, string buildPath)
 				var lines = File.ReadAllLines(path);
 				for(int i = 0 ; i < lines.Length ; i++)
 				{
-					if (lines[i].ToLower().Contains("nimbus"))
+					if (lines[i].ToLower().Contains("nimbussdk"))
```

Comment:
The Pod names are case sensitive; it is probably best to replace this with `lines[i].Contains("NimbusSDK")`